NEWS.md

```diff
@@ -10,6 +10,7 @@
 * provides `instance-storage-batch 1.1`
 * provides `instance-storage-batch-sync 1.1`
 * provides `inventory-view 1.1`
+* create instance optimization (MODINVSTOR-894)
```

Comment:
Please put this above the `provides` entries (which go last) and capitalize similarly to other new entries.

Reply:
updated.